Src/Base/AMReX_GpuAsyncArray.H

```diff
@@ -83,12 +81,9 @@ public:
 #if defined(AMREX_USE_HIP)
                 AMREX_HIP_SAFE_CALL ( hipStreamAddCallback(Gpu::gpuStream(),
                                                            amrex_asyncarray_delete, p, 0));
-#elif defined(__CUDACC__) && (__CUDACC_VER_MAJOR__ >= 10)
+#elif defined(AMREX_USE_CUDA)
```

Comment:
For clarity, we expect `__CUDACC__` and `AMREX_USE_CUDA` to be equivalent? The former is provided by the compiler, and we set the latter when we build, correct?

Reply:
Yes.  I used `AMREX_USE_CUDA` for consistence because the hip is tested with `AMREX_USE_HIP`.